### PR TITLE
add h5 featureMax

### PR DESCRIPTION
**Pull request by maximsondv**
feature max

### CODE REVIEW

File: git-practice/index.html [hunk 1]
```diff
@@ -9,5 +9,7 @@
 </head>
 <body>
 <h2>hello world</h2>
+<h5>Feature Max</h5>
```
Comment (ztimes2) [inline git-practice/index.html]: Would you pls use h4. h5 is ugly.

File: git-practice/index.html [hunk 2]
```diff
@@ -9,5 +9,7 @@
...
+<h3>Feature Max2</h3>
```
Comment (ztimes2) [inline git-practice/index.html]: How about making it "Feature Max 2" with extra space?

**Review by ztimes2 — CHANGES_REQUESTED** [review]

File: git-practice/index.html (hunk 1 above)
Comment (maximsondv) [reply git-practice/index.html]: > Would you pls use h4. h5 is ugly.
@ztimes2 ok bro

**Review by ztimes2 — APPROVED** [review]
LGTM

**Pull request merged**